Jenkinsfile

```diff
@@ -0,0 +1,108 @@
+#!groovy
+
+// Copyright 2017 Intel Corporation
```

Comment:
Do we need to update year to 2019? All other files has 2019

Reply:
Sure, will update to 2019